tensorbay/client/struct.py

```diff
@@ -456,15 +459,29 @@ class Draft(AttrsMixin, ReprMixin):
     branch_name: str = attr(key=camel)
     status: str = attr()
     description: str = attr(default="")
+    parent_commit_id: str = attr(key=camel)
+    author: User = attr()
+    updated_at: int = attr(key=camel)
 
     def __init__(  # pylint: disable=too-many-arguments
-        self, number: int, title: str, branch_name: str, status: str, description: str = ""
+        self,
+        number: int,
+        title: str,
+        branch_name: str,
+        status: str,
+        author: User,
+        updated_at: int,
+        parent_commit_id: str,
+        description: str = "",
     ) -> None:
         self.number = number
         self.title = title
         self.branch_name = branch_name
         self.status = status
         self.description = description
+        self.parent_commit_id = parent_commit_id
+        self.author = author
+        self.updated_at = updated_at
```

Comment:
the arguments order is inconsistent in three places, will it be better to make the order consistent?